entry.go

```diff
@@ -52,6 +64,7 @@ type TunaEntry struct {
 	bytesOut           uint64
 	bytesOutPaid       uint64
 	reverseBeneficiary common.Uint160
+	wallet             *nknsdk.Wallet
```

Comment:
If something exists in both entry and exit, we should try to use Common to avoid duplicate logic